.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "3rdparty/mshadow"]
 	path = 3rdparty/mshadow
-	url = https://github.com/dmlc/mshadow.git
+	url = https://github.com/DickJC123/mshadow.git
```

Comment:
temp reminder to update the mshadow submodule. It would be great if you could also refer the mshadow PR here.

Reply:
Don't merge this PR until the mshadow PR has been accepted:
https://github.com/dmlc/mshadow/pull/368.  The new mshadow functionality is tested in https://github.com/apache/incubator-mxnet/pull/13857.